java/distance-measurement-analysis/README.md

```diff
@@ -1,33 +1,33 @@
-and the ArcGIS API for JavaScript to your app. You can set the unit system of measurement (metric or imperial) and 
-have the units automatically switch to one appropriate for the current scale. The rendering is handled internally 
-so it doesn't interfere with other analyses like viewsheds.
+Measure distances between two points in 3D.
 
-![Distance Measurement Analysis App](distance-measurement-analysis.png)
+![Image of distance measurement analysis](distance-measurement-analysis.png)
+
+## Use case
+
+The distance measurement analysis allows you to add to your app the same interactive measuring experience found in ArcGIS Pro, City Engine, and the ArcGIS API for JavaScript. You can set the unit system of measurement (metric or imperial). The units automatically switch to one appropriate for the current scale.
 
 ## How To Use The Sample
-Choose an unit system for the measurement in the UI dropdown. Single tap any location on the screen to set the start point
-of measurement. Double tap and drag to set the end point of measurement. Single Tapping any new location will 
-start a new measurement. 
+Choose an unit system for the measurement in the UI dropdown. Single tap any location on the screen to set the start point of measurement. Double tap and drag to set the end point of measurement. Single Tapping any new location will start a new measurement.
+
+## How it works
 
-## How It Works
-To measure distances with the `LocationDistanceMeasurement` analysis:
+1. Create an `AnalysisOverlay` object and add it to the analysis overlay collection of the `SceneView` object.
+2. Specify the start location and end location to create a `LocationDistanceMeasurement` object. Initially, the start and end locations can be the same point.
+3. Add the location distance measurement analysis to the analysis overlay.
+4. The `measurementChanged` callback will trigger if the distances change. You can get the new values for the `directDistance`, `horizontalDistance`, and `verticalDistance` from the `MeasurementChangedEvent` object returned by the callback.
 
-1. Create an `AnalysisOverlay` and add it to your scene view's analysis overlay collection: `sceneView.getAnalysisOverlays().add(analysisOverlay)`. 
-1. Create a `LocationDistanceMeasurement`, specifying the `startLocation` and `endLocation`. These can be the same point to start with. Add the analysis to the analysis overlay: `analysisOverlay.getAnalyses().add(LocationDistanceMeasurement)`. The measuring line will be drawn for you between the two points. 
-1. The `measurementChanged` callback will fire if the distances change. You can get the new values for the `directDistance`, `horizontalDistance`, and `verticalDistance` from the `MeasurementChangedEvent` returned by the callback. The distance objects contain both a scalar value and a unit of measurement.
+## Relevant API  
 
-## Relevant API
 * AnalysisOverlay
 * LocationDistanceMeasurement
-* UnitSystem
+* MeasurementChangedEvent
```

Comment:
since this isn't a class or method in our API, let's remove it: it's fine to refer to it in the "How it works" section though since it's returned by the callback.